LottieDropper/Dropbox/DropboxBrowserViewModel.m

```diff
@@ -46,29 +49,43 @@ - (void)initializeDropboxClient: (void (^)(void)) done  {
 		NSLog(@" No Dropbox user client");
 		return;
 	}
-	[[self.client.filesRoutes listFolder:@""]
```

Comment:
Over stap 3 hadden jullie de discussie of dit wel nuttig was. Wel als je dit niet doet dan ziet de begruiker de animatie 2 keer. Of in het geval dat er geen eerdere lokale file is ziet hij even niets.